### PR TITLE
Added padding to protect against base64 decode failure

### PR DESCRIPTION
**Pull request by codewithsk**
*Issue #, if available:*
#425

*Description of changes:*
The padding bits in base64 encoded strings are removed during transit in the network. One way to recover from failure is to manually add padding bits to the end of the base64 encoded string before passing it to base64.decodestring(). For more information see: https://gist.github.com/perrygeo/ee7c65bb1541ff6ac770

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by lupesko** [comment]
@codewithsk thanks for submitting the PR.
There's a few things you will need to do:
- File an issue with an exact description of the bug: model used, request sent, exception raised and any thing else relevant
- Link the new issue to the PR. This will enable others to understand the issue and review it properly
- The PR is missing a unit test and an integration test to cover the issue - this will protect against future regressions, will help others understand the change you are making and will help you test your code.
- Build for the PR is failing - please make sure this passes the tests, otherwise the code will not be merged.

**Comment by codewithsk** [comment]
@lupesko thank you for taking the time to review this PR,

- An issue has been filed and the description of this PR has been edited to reference the issue. #423 
- Integration test has been added
- The build is currently failing on Python 3.5.x, but passes on Python 3.6. Currently working on patching the code to work on Python 3.5.x

File: load-test/run_load_test.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ if [ "$loops" == "" ]; then
 fi
 
 # make this var point to your installed CMDRunner location
-command_runner="/usr/local/Cellar/jmeter/3.3/libexec/lib/ext/CMDRunner.jar"
```
Comment (ankkhedia) [inline load-test/run_load_test.sh]: This is an unnecessary change as it just shows an example of path. It can be very different for users based on local path

Comment (codewithsk) [reply load-test/run_load_test.sh]: Oh looks like my working version got tagged on here. Reversed it back to 3.3

**Comment by codewithsk** [comment]
@ankkhedia Comment has been address and version reverted to 3.3 Thank you :+1:

**Comment by lupesko** [comment]
@ankkhedia comment addressed, merging.
